sdk/core/core-client-lro-rest/review/core-client-lro.api.md

```diff
@@ -20,6 +20,12 @@ export { LroEngineOptions }
 
 export { PollerLike }
 
+// @public
+export interface PollerOptions {
+    intervalInMs?: number;
```

Comment:
This is going to be customer-facing, so should it be `setIntervalInMs` to be consistent with what we have in Track 2 packages? I know our guidelines have them as `intervalInMs` but it looks like we wanted to adopt the convention of having a prefix verb so our packages now uses `setIntervalInMs`. @bterlson what do you think about this?

Reply:
Actually, code gen generates this: https://github.com/Azure/autorest.typescript/blob/357d655a4b8414490c90145aa98ea1130e691b05/test/integration/generated/lro/src/models/index.ts#L655